src/Sulu/Bundle/MarkupBundle/Markup/LinkTag.php

```diff
@@ -209,21 +214,24 @@ private function getContent(array $attributes)
     /**
      * @param mixed $href
      *
-     * @return array{uuid: string|null, anchor: string|null}
+     * @return array{uuid: string|null, query: string|null, anchor: string|null}
      */
-    private function getUuidAndAnchorFromHref($href): array
+    private function getPartsFromHref($href): array
     {
         $href = (string) $href ?: null;
 
         /** @var string[] $hrefParts */
         $hrefParts = $href ? \explode('#', $href, 2) : [];
+        $anchor = $hrefParts[1] ?? null;
 
+        $hrefParts = \explode('?', $hrefParts[0], 2);
```

Comment:
Some tests seems to fail here not sure in which cases `$hrefParts[0]` can not exist.

https://github.com/sulu/sulu/runs/4657974922?check_suite_focus=true